packages/@react-spectrum/menu/docs/Menu.mdx

```diff
@@ -295,19 +295,23 @@ within the Menu upon opening.
 
 ### Disabled
 
+To disable Menu items, pass an array of keys into the `disabledKeys` prop.
+If an item has both `key` and `id` attributes, the `key` attribute should be used for disabling.
+See the [Content](#events) section for more detail on how an item's `key` may be determined.
+
 ```tsx example
 <MenuTrigger>
   <ActionButton>
     Filter
   </ActionButton>
   <Menu
     items={[
-      {name: 'tiff', id: 'a1b2c3'},
-      {name: 'png', id: 'g5h1j9'},
-      {name: 'jpg', id: 'p8k3i4'},
-      {name: 'PDF', id: 'j7i3a0'}
+      {name: 'tiff', key: 'tiff', id: 'a1b2c3'},
```

Comment:
I don't think we should encourage people to add both a key and an id to their objects. I'd rather keep the examples showing what you should do rather than what you shouldn't. Maybe keep the example the same as it was and add the description above clarifying?

Reply:
I would agree here. Part of what confused me was I thought I _had_ to use an id for disabling items, even though I was already using a `key`. I also added `id`s to each item object as a result. I think most people will have keys on their items so that they can attach actions to them. Personally, I think things would have been clearer if just `key` was used rather than id. Then maybe mention that you can use `id` in place of `key` if needed. Just one person's thoughts! You guys rock.